### ISSUE TITLE
/gearmenu opt does not display

### ISSUE THREAD

**Issue by wcj8810** [RagedUnicorn/wow-classic-gearmenu#97]
The /gearmenu opt command does not display the options window. 

The below LUA error is captured from BugSack.

```
31x GearMenu/code/GM_Cmd.lua:64: attempt to call field 'OpenAddonPanel' (a nil value)

[string "@GearMenu/code/GM_Cmd.lua"]:64: in function `?'

[string "@FrameXML/ChatFrame.lua"]:4966: in function <FrameXML/ChatFrame.lua:4912>

[string "=[C]"]: ?

[string "=[C]"]: in function `ChatEdit_ParseText'

[string "@FrameXML/ChatFrame.lua"]:4629: in function <FrameXML/ChatFrame.lua:4628>

[string "=[C]"]: in function `ChatEdit_SendText'

[string "@FrameXML/ChatFrame.lua"]:4665: in function `ChatEdit_OnEnterPressed'

[string "*ChatFrame.xml:121_OnEnterPressed"]:1: in function <[string "*ChatFrame.xml:121_OnEnterPressed"]:1>

 

Locals:

msg = "opt"

args = <table> {

 1 = "opt"

}

(*temporary) = nil

(*temporary) = "attempt to call field 'OpenAddonPanel' (a nil value)"

mod = <table> {

 gearBar = <table> {

 }

 gearBarConfigurationMenu = <table> {

 }

 addonConfiguration = <table> {

 }

 configuration = <table> {

 }

 cmd = <table> {

 }

 gearBarChangeMenu = <table> {

 }

 target = <table> {

 }

 aboutContent = <table> {

 }

 gearBarManager = <table> {

 }

 common = <table> {

 }

 macro = <table> {

 }

 tooltip = <table> {

 }

 Initialize = <function> defined @GearMenu/code/GM_Core.lua:224

 quickChangeMenu = <table> {

 }

 themeClassic = <table> {

 }

 combatQueue = <table> {

 }

 itemManager = <table> {

 }

 keyBind = <table> {

 }

 gearManager = <table> {

 }

 libUIDropDownMenuTemplates = <table> {

 }

 themeCustom = <table> {

 }

 L = <table> {

 }

 OnLoad = <function> defined @GearMenu/code/GM_Core.lua:52

 uiHelper = <table> {

 }

 logger = <table> {

 }

 generalMenu = <table> {

 }

 RegisterEvents = <function> defined @GearMenu/code/GM_Core.lua:61

 ShowWelcomeMessage = <function> defined @GearMenu/code/GM_Core.lua:257

 gearBarConfigurationSubMenu = <table> {

 }

 themeCoordinator = <table> {

 }

 gearBarStorage = <table> {

 }

 OnEvent = <function> defined @GearMenu/code/GM_Core.lua:100

 quickChange = <table> {

 }

 ticker = <table> {

 }

 libUiDropDownMenu = <table> {

 }

 filter = <table> {

 }

 trinketMenu = <table> {

 }

 trinketConfigurationMenu = <table> {

 }

 cooldown = <table> {

 }

 tag = "Core"

}

me = <table> {

 SetupSlashCmdList = <function> defined @GearMenu/code/GM_Cmd.lua:45

 tag = "Cmd"

}

ShowInfoMessage = <function> defined @GearMenu/code/GM_Cmd.lua:36

```

**Comment by RagedUnicorn** [comment]
Hi @wcj8810,

Thanks for bringing this up. There is indeed an error here. I will fix it with the next release (should be possible during the weekend)

Cheers,
RagedUnicorn

**Comment by RagedUnicorn** [comment]
Fixed with 2.3.2. Thank you for reporting this.
